website/docs/d/team.html.markdown

```diff
@@ -25,17 +25,27 @@ data "mongodbatlas_teams" "test" {
 	org_id     = mongodbatlas_teams.test.org_id
 	team_id    = mongodbatlas_teams.test.team_id
 }
+
+data "mongodbatlas_teams" "test2" {
+	org_id     = mongodbatlas_teams.test.org_id
+	name       = mongodbatlas_teams.test.name
+}
 ```
 
+
 ## Argument Reference
 
 * `org_id` - (Required) The unique identifier for the organization you want to associate the team with.
-* `team_id` - (Required) The unique identifier for the team.
+* `team_id` - (Optional) The unique identifier for the team.
+* `name` - (Optional) The team name.
+
+~> **IMPORTANT:** Either `team_id` or `name` must be configurated.
 
 
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
+
 * `id` -	The Terraform's unique identifier used internally for state management.
 * `name` -  The name of the team you want to create.
```

Comment:
Note that name is put here has a computed but your schema says that it's not, as we will change the `name` and `team_id` as computed, we need to put `team_id` here